selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -655,6 +656,22 @@ public void goToPosition(int positionLine, int positionChar) {
     waitCursorPosition(positionLine, positionChar);
   }
 
+  /**
+   * Sets cursor to specified {@code positionLine} and {@code positionChar} and checks result.
+   *
+   * @param positionLine line's number where cursor should be placed
+   * @param positionChar char's number where cursor should be placed
+   */
+  public void selectLines(int fromLine, int numberOfLine) {
+    Actions action = actionsFactory.createAction(seleniumWebDriver);
+    setCursorToLine(fromLine);
+    action.keyDown(SHIFT).perform();
+    for (int i = 0; i < numberOfLine; i++) {
+      typeTextIntoEditor(Keys.ARROW_DOWN.toString());
```

Comment:
In this case, if cursor in the editor placed to the beginning of the row the last row will be not selected 
![ddddd](https://user-images.githubusercontent.com/22724619/44774683-61ce3b00-ab7c-11e8-9688-1886192a53b7.png)